### ISSUE TITLE
.git/crypt/transcrypt: line 245: GIT_REFLOG_ACTION: unbound variable

### ISSUE THREAD

**Issue by ZhymabekRoman** [elasticdog/transcrypt#150]
Thank you for making transcrypt! When applying a stash with the `git stash pop` command, this error appears:
```
.git/crypt/transcrypt: line 245: GIT_REFLOG_ACTION: unbound variable
```
transcrypt version: https://github.com/elasticdog/transcrypt/commit/e08c3595e6ec57bc448e50401cb7ed845d866419
git version: 2.30.2

**Comment by jmurty** [comment]
Hi @ZhymabekRoman I haven't been able to reproduce this error with some basic testing of `git stash` and `git stash pop` but I have applied a potential fix to the one place Transcrypt refers to `GIT_REFLOG_ACTION` in commit bcc46694

Can you test the latest version on the `main` branch and confirm whether it fixes the `unbound variable` error for you?

**Comment by ZhymabekRoman** [comment]
@jmurty, Thanks for reply! 
> I haven't been able to reproduce this error with some basic testing

I think it's because I forgot to mention Auto-merging during the `git stash`.

Full command output with latest `transcrypt` version:
```bash
roman@debian:~/Obsidian$ git stash pop
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.
.git/crypt/transcrypt: line 267: THEIRS_LABEL: unbound variable
Auto-merging diary/.obsidian/plugins/recent-files-obsidian/data.json
CONFLICT (content): Merge conflict in diary/.obsidian/plugins/recent-files-obsidian/data.json
The stash entry is kept in case you need it again.
```

**Comment by jmurty** [comment]
Hi @ZhymabekRoman thanks for that clarification. I think I've fixed it properly now in commit 155808d9, can you please re-test to confirm?

To be clear for anyone following along, or reading this later, this bug was triggered when you:
- Stash changes to a secret file (A)
- Make further changes to that same secret file (A')
- Apply or pop the stash *after* adding/staging the A' changes to git's index

If the further changes are staged, git will try to merge the stash content instead of failing with the usual `error: Your local changes to the following files would be overwritten by merge` error.

Despite using git for years, I didn't know that staging file changes would trigger auto-merge behaviour when applying a stash with changes to the same file. Hat tip to https://stackoverflow.com/a/16613814 for teaching me this.

**Comment by ZhymabekRoman** [comment]
@jmurty, I tested the latest version of transcrypt and everything seems to work perfectly! Thank you!
